src/lib/pagination/ctrl.coffee

```diff
@@ -0,0 +1,99 @@
+EventsEmitter = require( 'events' )
+Pagination = require( './index' )
+PaginationView = require( './view' )
+
+###
+ * #PaginationCtrl
+ * Creates a pagination element and controls it.
+ *
+ * ## Events
+ *
+ * | CONSTANT.argument  | Description |
+ * |--------------------| -------------|
+ * | CHANGE | When page number is changed by user |
+ * | CHANGE.current  | New page number |
+ * | CHANGE.last | Number of last page |
+###
+class PaginationCtrl extends EventsEmitter
```

Comment:
I see this like the following:
We have a class "David" and a class "Telephone".

Telephone can be used by David... but should David extend from telephone? isn't it "strange"?
I would create a Person class that implements the abstract methods that uses the telephone, so David (or any person) can extend the Person class and has access to it...

I'm not saying that you have to create another class (to avoid overcomplexity), but still, I don't see a controller extending from a class like EventEmitter. I rather prefer to pass (or create) it on the constructor.

But of course It's just my point of view.

P.D: My example sucks, I know.